src/functionalTest/java/uk/gov/hmcts/darts/dailylist/DailylistFunctionalTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.hmcts.darts.dailylist;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.darts.FunctionalTest;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class DailylistFunctionalTest extends FunctionalTest {
+
+    public static final String POST_DAILYLIST_URL = "/dailylists";
+
+    public static final String POST_DAILYLIST_RUN_URL = "/dailylists/run";
+
+    @Test
+    void postDailyList() throws IOException {
+        String todayDateString = LocalDate.now().toString();
+        String tomorrowDateString = LocalDate.now().plusDays(1).toString();
+
+        String xmlDocument = getContentsFromFile("DailyList-Document.xml");
+
+        Response response = buildRequestWithAuth()
+            .contentType(ContentType.JSON)
+            .queryParam("source_system", "XHB")
+            .queryParam("courthouse", "Swansea")
+            .queryParam("hearing_date", tomorrowDateString)
+            .queryParam("unique_id", "1111111")
+            .queryParam("published_ts", todayDateString + "T23:30:52.123Z")
+            .header("xml_document", xmlDocument)
+            .when()
+            .baseUri(getUri(POST_DAILYLIST_URL))
+            .redirects().follow(false)
+            .post().then().extract().response();
+
+        assertEquals(200, response.getStatusCode());
+
+        Integer dalId = response.jsonPath().get("dal_id");
+
+        String jsonDocument = getJsonDocumentWithValues(todayDateString, tomorrowDateString);
+
+        //then patch it with JSON
+        response = buildRequestWithAuth()
+            .contentType(ContentType.JSON)
+            .queryParam("dal_id", dalId)
+            .header("json_document", jsonDocument)
+            .when()
+            .baseUri(getUri(POST_DAILYLIST_URL))
+            .redirects().follow(false)
+            .patch().then().extract().response();
+
+        assertEquals(200, response.getStatusCode());
+
+        Integer cthId = 1;
+
+        //process dailylist
+        response = buildRequestWithAuth()
```

Comment:
I'm not sure this adds much to the test, the functionality is really to create the daily list document, with XML and JSON. The running/processing of it comes later via a scheduled job.